src/dotnet/commands/dotnet-test/LocalizableStrings.cs

```diff
@@ -10,40 +10,40 @@ internal class LocalizableStrings
 
         public const string CmdArgDescription = "The project to test, defaults to the current directory.";
 
-        public const string CmdSettingsFile = "SettingsFile";
+        public const string CmdSettingsFile = "SETTINGS_FILE";
 
         public const string CmdSettingsDescription = "Settings to use when running tests.";
 
         public const string CmdListTestsDescription = @"Lists discovered tests";
 
-        public const string CmdTestCaseFilterExpression = "Expression";
+        public const string CmdTestCaseFilterExpression = "EXPRESSION";
 
         public const string CmdTestCaseFilterDescription = @"Run tests that match the given expression.
                                         Examples:
                                         --test-case-filter:""Priority = 1""
                                         --test-case-filter: ""(FullyQualifiedName~Nightly | Name = MyTestMethod)""";
 
         public const string CmdTestAdapterPathDescription = @"Use custom adapters from the given path in the test run.
-                                        Example: --test-adapter-path:<pathToCustomAdapters>";
+                                        Example: --test-adapter-path <PATH_TO_ADAPTER>";
 
         public const string CmdLoggerOption = "LoggerUri/FriendlyName";
 
         public const string CmdLoggerDescription = @"Specify a logger for test results. 
                                         Example: --logger:trx";
 
-        public const string CmdConfiguration = "configuration";
+        public const string CmdConfiguration = "CONFIGURATION";
 
-        public const string CmdConfigDescription = @"Configuration under which to build, i.e. Debug/Release";
+        public const string CmdConfigDescription = "Configuration to use for building the project.  Default is  \"Debug\".";
```

Comment:
This has appeared over and over again. Feels to me like we should have this one in a CommonLocalizableStrings or something like that.